.github/workflows/build-git-installers.yml

```diff
@@ -764,6 +804,10 @@ jobs:
       - name: Install macOS
         if: contains(matrix.component.os, 'macos')
         run: |
+          # avoid letting Homebrew's `git` in `/opt/homebrew/bin` override `/usr/local/bin/git`
+          test arm64 != "$(uname -m)" ||
```

Comment:
Super nitpicky, but why not use `test x86_64` as you did above?

Reply:
Honestly, I'm not quite sure what is more correct: Is `/opt/homebrew/` used because it's ARM64? Or is `/usr/local/` only used for x86_64? So far, we only have those two CPU architectures to deal with, which means that `test arm64 !=` and `test x86_64 =` would both work, but... 😄